rust/worker/src/config.rs

```diff
@@ -573,6 +629,14 @@ mod tests {
                     blockfile_provider:
                         Arrow:
                             max_block_size_bytes: 16384
+                            block_manager_config:
```

Comment:
we will have to remember to update the config in staging when we merge this otherwise staging will crash